### PR TITLE
fix(dailyInterval): misleading description

### PR DESCRIPTION
**Pull request by gxrsti**
Changed the description of the daily messages interval modal to be more descriptive.

### CODE REVIEW

File: src/buttons/dailyInterval.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ const modalObject = {
           type: 4,
           style: 1,
           custom_id: "input",
-          label: "Enter a 24 hour dailymsg interval (HH:MM).",
+          label: "At what time should the daily message be posted? (HH:MM)",
```
Comment (mezotv) [inline src/buttons/dailyInterval.js]:
```suggestion
          label: "When should the daily message be posted? (HH:MM)",
```
Perhaps this could be better?

**Review by mezotv — CHANGES_REQUESTED** [review]
For this to be merged it should also edit the button labels around the bot as well as translations/titles

File: src/buttons/dailyInterval.js (hunk 1 above)
Comment (gxrsti) [reply src/buttons/dailyInterval.js]: Which buttons are you referring to?

Comment (mezotv) [reply src/buttons/dailyInterval.js]: > Which buttons are you referring to?

For example the buttons in the settings menu that open this modal.

Comment (gxrsti) [reply src/buttons/dailyInterval.js]: So you want to rename the "daily interval" completely? What do you want to call it?

My approach would be "time"/"daily messages time".

Also do you want to write "message" or "messages"? (Shouldn't it be singular?)

Comment (mezotv) [reply src/buttons/dailyInterval.js]: Thats a good question maybe "Post time" works.

Comment (gxrsti) [reply src/buttons/dailyInterval.js]: > Also do you want to write "message" or "messages"? (Shouldn't it be singular?)

Any thoughts on this one?

Comment (mezotv) [reply src/buttons/dailyInterval.js]: Can be singular as its only one message yes

**Review by mezotv — APPROVED** [review]
Did some changed to the text should look good now thanks for your pr!

**Pull request merged**